drivers/periph_common/Kconfig.timer

```diff
@@ -30,3 +30,10 @@ config MOD_PERIPH_INIT_TIMER_PERIODIC
     default y if MOD_PERIPH_INIT
 
 endif # MOD_PERIPH_TIMER
+
+menuconfig KCONFIG_MODULE_PERIPH_TIMER
+    bool "Timer peripheral driver"
+    depends on MODULE_PERIPH_TIMER
+
+# Include CPU specific configurations
+osource "$(RIOTCPU)/$(CPU)/periph/Kconfig.timer"
```

Comment:
These plaftorm specific options should be shown only when `KCONFIG_MODULE_PERIPH_TIMER` is active right? We can do the following, so we don't have to specify the dependency on every symbol inside the file:
```suggestion
# Include CPU specific configurations
if KCONFIG_MODULE_PERIPH_TIMER
osource "$(RIOTCPU)/$(CPU)/periph/Kconfig.timer"
endif
```
**Note**: This is not a conditional inclusion of the file. This is applying the dependency to all the symbols defined inside.

Reply:
addressed